### ISSUE TITLE
Installation problem - The option `nixpkgs.localSystem' defined in `<unknown-file>' does not exist.

### ISSUE THREAD

**Issue by a-h** [LnL7/nix-darwin#217]
I'm having some issues (re)-installing nix-darwin. I removed nix `sudo rm -rf /nix` and reinstalled it:

```
sh <(curl -L https://nixos.org/nix/install) --darwin-use-unencrypted-nix-store-volume
. /Users/adrian/.nix-profile/etc/profile.d/nix.sh
```

I'm able to use the nix commands as usual, `nix-env -e hello` etc, and I'm able to build the installer:

```
nix-build https://github.com/LnL7/nix-darwin/archive/master.tar.gz -A installer      
```

However, when I then run it...

```
# Have to reference diskutil (maybe the installer needs updating for that?)
export PATH=$PATH:/usr/sbin/
./result/bin/darwin-installer
```

I get an error. I've moved my dotfiles and configuration out of the way, so it's just attempting to use the default configuration, and I've even removed my `.zshrc`

```
cat ~/.nixpkgs/darwin-configuration.nix
```

```nix
{ config, pkgs, ... }:

{
  # List packages installed in system profile. To search by name, run:
  # $ nix-env -qaP | grep wget
  environment.systemPackages =
    [ pkgs.vim
    ];

  # Use a custom configuration.nix location.
  # $ darwin-rebuild switch -I darwin-config=$HOME/.config/nixpkgs/darwin/configuration.nix
  # environment.darwinConfig = "$HOME/.config/nixpkgs/darwin/configuration.nix";

  # Auto upgrade nix package and the daemon service.
   services.nix-daemon.enable = true;
   nix.package = pkgs.nix;

  # Create /etc/bashrc that loads the nix-darwin environment.
  programs.zsh.enable = true;  # default shell on catalina
  # programs.fish.enable = true;

  # Used for backwards compatibility, please read the changelog before changing.
  # $ darwin-rebuild changelog
  system.stateVersion = 4;
}
```

```
Installing nix-darwin...

Would you like edit the default configuration.nix before starting? [y/n] n
warning: Nix search path entry '/nix/var/nix/profiles/per-user/root/channels' does not exist, ignoring
error: while evaluating the attribute 'activationScript' of the derivation 'darwin-system-20.09pre238994.078b9f8aa68+darwin4.0000000' at /nix/store/aiqw7lmqbnbdq33za1d0qbmxid05zjj3-master.tar.gz/modules/system/default.nix:74:7:
while evaluating the attribute 'activationScripts.script.text' at undefined position:
while evaluating 'g' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/attrsets.nix:276:19, called from undefined position:
while evaluating anonymous function at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:98:72, called from /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/attrsets.nix:279:20:
while evaluating the attribute 'value' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:450:9:
while evaluating the option `system.activationScripts.script.text':
while evaluating the attribute 'mergedValue' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:482:5:
while evaluating the attribute 'values' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:476:9:
while evaluating the attribute 'values' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:575:7:
while evaluating anonymous function at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:462:28, called from /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:462:17:
while evaluating definitions from `/nix/store/aiqw7lmqbnbdq33za1d0qbmxid05zjj3-master.tar.gz/modules/system/activation-scripts.nix':
while evaluating 'dischargeProperties' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:534:25, called from /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:463:137:
while evaluating the attribute 'value' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:327:44:
while evaluating the attribute 'activationScripts.applications.text' at undefined position:
while evaluating 'g' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/attrsets.nix:276:19, called from undefined position:
while evaluating anonymous function at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:98:72, called from /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/attrsets.nix:279:20:
while evaluating the attribute 'value' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:450:9:
while evaluating the option `system.activationScripts.applications.text':
while evaluating the attribute 'mergedValue' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:482:5:
while evaluating the attribute 'values' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:476:9:
while evaluating the attribute 'values' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:575:7:
while evaluating anonymous function at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:462:28, called from /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:462:17:
while evaluating definitions from `/nix/store/aiqw7lmqbnbdq33za1d0qbmxid05zjj3-master.tar.gz/modules/system/applications.nix':
while evaluating 'dischargeProperties' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:534:25, called from /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:463:137:
while evaluating the attribute 'value' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:327:44:
while evaluating the attribute 'passAsFile' of the derivation 'system-applications' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/pkgs/build-support/trivial-builders.nix:7:7:
while evaluating the attribute 'buildCommand' of the derivation 'darwin-manpages' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/pkgs/build-support/trivial-builders.nix:7:7:
while evaluating the attribute 'buildCommand' of the derivation 'manual-olinkdb' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/pkgs/build-support/trivial-builders.nix:7:7:
while evaluating the attribute 'buildCommand' of the derivation 'darwin-manual-combined' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/pkgs/build-support/trivial-builders.nix:7:7:
while evaluating the attribute 'buildCommand' of the derivation 'generated-docbook' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/pkgs/build-support/trivial-builders.nix:7:7:
while evaluating the attribute 'buildCommand' of the derivation 'options-db.xml' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/pkgs/build-support/trivial-builders.nix:7:7:
while evaluating 'flip' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/trivial.nix:138:16, called from /nix/store/aiqw7lmqbnbdq33za1d0qbmxid05zjj3-master.tar.gz/doc/manual/default.nix:48:21:
while evaluating 'optionAttrSetToDocList'' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/options.nix:140:37, called from /nix/store/aiqw7lmqbnbdq33za1d0qbmxid05zjj3-master.tar.gz/doc/manual/default.nix:9:72:
while evaluating 'collect' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/attrsets.nix:178:19, called from /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/options.nix:162:67:
while evaluating 'isType' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/types.nix:14:18, called from /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/attrsets.nix:179:8:
while evaluating 'evalModules' at /nix/store/8bjbkgp828zhcxhmwm88w71vap1jwipw-nixpkgs-20.09pre238994.078b9f8aa68/nixpkgs/lib/modules.nix:21:17, called from /nix/store/aiqw7lmqbnbdq33za1d0qbmxid05zjj3-master.tar.gz/modules/documentation/default.nix:19:24:
The option `nixpkgs.localSystem' defined in `<unknown-file>' does not exist.
```

Any ideas where this error is coming from, or how to troubleshoot it?

**Comment by a-h** [comment]
Could be related to https://github.com/LnL7/nix-darwin/pull/216

**Comment by a-h** [comment]
It is. A temporary workaround is to disable documentation until it's merged in your darwin-configuration.nix

```nix
{
  ...
  # Used for backwards compatibility, please read the changelog before changing.
  # $ darwin-rebuild changelog
  system.stateVersion = 4;

  # Disable documentation until https://github.com/LnL7/nix-darwin/issues/217 is fixed.
  documentation.enable = false;
}
```

**Comment by LnL7** [comment]
Fixed in 58e6ca77d92d400c23609a27462547438b1122a5
